### DISCUSSION TITLE
Trips missing

### DISCUSSION THREAD

**Discussion by pixelchrome** [teslamate-org/teslamate#1365] [Q&A]
From time to time I see that trips not showing up in teslamate. How can I improve that?

Example: I am missing a whole day with around 230 km.

**Comment by snicoll** [comment]
What's the state of the car in the Overview panel for that period? It could happen that the car is flagged offline (which means data wasn't logged at all). This happens to me from time to time when the network goes down.

**Reply by pixelchrome** [reply]
See screenshot below. Network shouldn't be a problem. We've started our trip at around 7.30... 1st drive was until 8.30. Charging at the parking lot. 2nd drive was back home... (12.30 - 14.00).

During that time it was still in status charging...

![garfana_overview](https://user-images.githubusercontent.com/10322584/108592791-ec6c0b80-736f-11eb-9b3e-3ef3a5020b14.png)

**Reply by pixelchrome** [reply]
See the flat line between 4.46 and 14.04...

**Reply by pixelchrome** [reply]
My logs are no longer available :-(

**Comment by adriankumpf** [comment]
The fact that a whole day is missing is odd. Take a look at the log output for the respective points in time (if they are still available): `docker-compose logs teslamate`

**Reply by cyberlauncher** [reply]
I've also been having issues with missing drives and other details from my installation dating back the the 1st of February.
No new drives are showing. I used the car yesterday at approximately 10:00-11:00 . Immediately after the drive a charge session was initiated yet nothing shows except it being asleep.
I logged out of Teslamate and logged back in with no changes in status. I had aslo brought down the docker app and restarted it.
There is no substantial information that I can glean from the Teslamate logs.
Teslamate ver 2021.4.3
![2021-03-02 10_00_46-Overview - Grafana — Mozilla Firefox](https://user-images.githubusercontent.com/14206985/109693642-c2b89e80-7b3e-11eb-8728-0b6673895034.png)

**Reply by BrandX3** [reply]
I'm having similar issues. On the drives page, the last recorded drive is from two days ago. On the overview page, it shows that I was driving today from 7:01 to 13:10, nonstop. Actually, I drove from 7:01 until about 8:00, then a couple of short drives at around 8:20 (outbound) and 9:00 (back home), then a longer outbound at 9:30 until about 10:10, and then back home from about 11:15 until 12:00. I went out again at 13:27 and I was home by 14:00. The car sat until I drove about 50 meters and started charging at 16:00.
![teslamate](https://user-images.githubusercontent.com/10144757/111053684-a825d580-841a-11eb-856a-7296148b6e9f.png)
